ecs-agent/tcs/model/api/api-2.json

```diff
@@ -120,7 +121,8 @@
       "members":{
         "containerName":{"shape":"String"},
         "healthStatus":{"shape":"HealthStatus"},
-        "statusSince":{"shape":"Timestamp"}
+        "statusSince":{"shape":"Timestamp"},
+        "statusMessage":{"shape":"String"}
```

Comment:
Xing, What is this new field for?

Reply:
this field is added over 2 years ago per the TACS model code. But not sure why we didn't see this until now. Probably we didn't update the TACS for long time.

Reply:
Not sure. I recently updated the TACS model to include ephemeralStorageMetrics. Are we using this field while sending the ContainerHealth messages?

Reply:
> Are we using this field while sending the ContainerHealth messages?

Looks yes.